frontend/js/leaderboard.js

```diff
@@ -116,28 +122,23 @@ class Leaderboard {
     runDeleteButton.textContent = 'Delete';
     runDeleteButton.classList.add("btn", "btn-primary", "link");
 
-    runContainer.append(userName, runScore, runWordsTyped, runWordsSeen, runDeleteButton);
+    runContainer.append(userName, runScore, runWordsTyped, runWordsSeen);
 
-    //BUG: Current user can delete ANY user run 
-    //TODO: Add figure out global account_id for curent user
-    // if (game.account === account.account_id) {
-    //   runContainer.append(runDeleteButton);
-    // }
+    if (game.account !== undefined && Number(game.account.id) === Number(account.id)) {
```

Comment:
would game.account evaluate to truthy? That way we don't have to compare it to undefined